package test.work.test.work.exceptions;

public class NotExistsWithUsernameException extends Throwable {
    public NotExistsWithUsernameException(String message) {
        super(message);
    }
}
